solarforecastarbiter/validation/validator.py

```diff
@@ -262,6 +262,27 @@ def check_wind_limits(weather, wind_limits=(0., 60.)):
     return flags
 
 
+def check_rh_limits(rh, rh_limits=(5, 100)):
```

Comment:
I suggest 0 for the lower bound. 5% is rare, but not unreasonable. I don't think it hurts to allow RH to drop to 0.

Reply:
From SPC sounding climatology page https://www.spc.noaa.gov/exper/soundingclimo/

<img width="894" alt="Screen Shot 2019-04-23 at 2 52 37 PM" src="https://user-images.githubusercontent.com/4383303/56618437-7bbf0100-65d7-11e9-9250-dd7e0216b3ad.png">